.github/workflows/linux_meson.yml

```diff
@@ -276,10 +276,6 @@ jobs:
         sudo apt-get update
         sudo apt-get install -y libgmp-dev libmpfr-dev libmpc-dev ccache gfortran
 
-        # Install OpenBLAS a la cibuildwheel.
-        chmod +x tools/wheels/cibw_before_build_linux.sh
-        sudo tools/wheels/cibw_before_build_linux.sh --nightly .       
```

Comment:
I think this was getting a nightly openblas build, it doesn't look as if a nightly scipy-openblas wheel is being installed though.

Reply:
Ah good catch, I overlooked that. We have nightlies at https://anaconda.org/scientific-python-nightly-wheels/scipy-openblas32.